test/lib/ansible_test/_util/controller/sanity/yamllint/yamllinter.py

```diff
@@ -164,7 +164,6 @@ def get_module_docs(self, path, contents):  # type: (str, str) -> t.Dict[str, t.
         """Return the module documentation for the given module contents."""
         module_doc_types = [
             'DOCUMENTATION',
-            'EXAMPLES',
```

Comment:
This is a major change IMO and should not happen without a changelog fragment and sneaked into a docs PR.

Reply:
(Also I think this change should not happen. The mechanism described in https://github.com/ansible/ansible/pull/82355#issuecomment-1841469933 should be used instead.)

Reply:
agreed, i was unaware of such